src/apps/Login/components/FormInput.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const FormInput = (props) => {
+
+  const {label, inputType, inputName, placeholder, onChange } = props;
+
+  return (
+    <div className="form-group">
+      {label}
+      <input type={inputType} onChange={onChange} className="form-control" name={inputName} placeholder={placeholder} required />
+    </div>
+  );
+};
+
+export const utilsPropTypes = {
+  inputType: PropTypes.string.isRequired,
+  inputName: PropTypes.string.isRequired,
+  placeholder: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired
+};
+
+FormInput.propTypes = utilsPropTypes;
+
```

Comment:
Hi. Awesome work on getting the work done. Can I request a minor change that you have this file on the common components folder so that we can have it reusable with our features too since we have the forms with the same input fields etc. It'll help a lot since we won't have to redefine this again on our side.